pkg/native/unserializers/unserializer_cdx.go

```diff
@@ -28,8 +28,8 @@ func NewCDX(version, encoding string) *CDX {
 	}
 }
 
-// ParseStream reads a CycloneDX 1.5 from stream r usinbg the offcial CycloneDX
-// libraries and returns a protobom document with its data.
+// Unserialize reads datq data from io.Reader r and parses it as a CycloneDX
```

Comment:
```suggestion
// Unserialize reads data from io.Reader r and parses it as a CycloneDX
```

Reply:
I'll fix this typo in another PR to avoid pushing a commit just for this (I can't rebase cleanly as the branch now has Jonathan's commits)

Reply:
🤦‍♂️ sorry!!! lol